x-pack/plugins/fleet/public/applications/fleet/sections/settings/components/edit_output_flyout/index.tsx

```diff
@@ -138,7 +138,17 @@ export const EditOutputFlyout: React.FunctionComponent<EditOutputFlyoutProps> =
               isLogstashOutput && (
                 <FormattedMessage
                   id="xpack.fleet.editOutputFlyout.logstashTypeOutputBetaHelpText"
-                  defaultMessage="Logstash output is in beta. Please send feedback to report bugs or suggest improvements."
+                  defaultMessage="Logstash output is in beta. Click {sendFeedback} to report bugs and suggest improvements."
```

Comment:
Is this the right way to make text bold? Seems a bit cumbersome....

Reply:
Yes it's the correct way to do it, we are using an old version of the `react-intl` library that do not allow to have short tag like `Logstash output is in beta. Please <b>send feedback</b> to report bugs or suggest`

Reply:
Where is the scream emoji when you need it? Luckily, I guess, most text strings should not have special formatting.